.github/workflows/docker-build-push-release.yml

```
@@ -25,7 +25,7 @@ jobs:
         uses: docker/build-push-action@v2
```

actionlint (1.7.8)

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)